README.md

```diff
@@ -1,5 +1,8 @@
 # Nyrna
 
+[![Packaging status](https://repology.org/badge/tiny-repos/nyrna.svg)](https://repology.org/project/nyrna/versions)
+[![latest packaged version(s)](https://repology.org/badge/latest-versions/nyrna.svg)](https://repology.org/project/nyrna/versions)
```

Comment:
I think it looks better without these two, the verticle one under `# Installing` looks better too.
```suggestion
```